### DISCUSSION TITLE
Output in table

### DISCUSSION THREAD

**Discussion by oflavioc** [MikeMeliz/TorCrawl.py#9] [Ideas]
Works like a charm! Would be great to have the output in a user friendly format, like XML or JASON, or maybe a simple DB with a table showing link, references like subdomains, etc.. Great job.

**Comment by MikeMeliz** [comment]
Hey, thanks! 

That's a nice idea, it would be great with arguments like `--db` or `--json`!

I could give it a try first with SQLite to store them at a database, and it'll be pretty simple later to return them as JSON.
 
Though, would you give me an example of a database? Like, a simple database for each domain and tables like `Links`, `Subdomains`, `Mails` and so on? Or something more complete, like a table of root domains and the rest of the findings will point out to those domains -with foreign keys etc?

**Reply by oflavioc** [reply]
I wasn't looking for anything fancy, just a simple database for each domain, organized with line numbers. I heard about a way to combine the output using a Pandas Dataframe, might be a good idea to do the trick.

**Reply by MikeMeliz** [reply]
Oh, I didn't know about this dataframe, seems really cool and simple! I'll give it a try soon and I'll let you know!

**Comment by MikeMeliz** [comment]
Closing this (super old) discussion, as export have been finally improved with [the latest release](https://github.com/MikeMeliz/TorCrawl.py/releases/tag/v1.35):
- Export in (linear) JSON and XML
- Export in SQLite3 with links between URLs.
